changelog/d_boundscheck.dd

```diff
@@ -0,0 +1,29 @@
+Add version identifier `D_BoundsChecks`
+
+One stylistic rule of D is that version identifier should be positive.
+`D_NoBoundsChecks` stands out as an exception in this regard.
+This version adds the opposite `D_BoundsChecks` so that code have at least
```

Comment:
I think we might be subject to the deprecation policy being proposed at https://github.com/dlang/DIPs/blob/master/DIPs/DIP1013.md